jit-compiler/src/lib.rs

```diff
@@ -70,7 +71,8 @@ pub fn compile<T: FieldElement>(
 
     if successful_symbols.len() < requested_symbols.len() {
         let successful_hash = successful_symbol_names.iter().collect::<HashSet<_>>();
-        log::info!(
+        // TODO this should be changed back to Info after the introduction of the ToCol trait.
+        log::debug!(
```

Comment:
this one generates quite a large message because of all symbols that didn't work